### PR TITLE
Update test images

### PR DESCRIPTION
**Pull request by leena-jawale**
# Description
Updates test VM images for cve release

## PR task checklist

- [ ] I have performed a self-review of my code
- [ ] I have followed code style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added thorough tests
- [ ] I have had a successful deployment with the changes in this PR

# Jira

This PR is for JIRA item: <https://issues.redhat.com/browse/AAP-NNNN>

# Testing

## Steps to test
1. Pull down the PR
2. ...
3. ...

## Expected result

With the changes in this PR you will see ...

### CODE REVIEW

File: images/gcp/vm_images.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-gcp_foundation_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e1_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e2_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e4_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-image_internal_install_name: quay.io/ansible-on-clouds/ansible-on-clouds-internal-install:2.3-latest
\ No newline at end of file
+gcp_foundation_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-main-gcp-dev
```
Comment (chrisahl) [inline images/gcp/vm_images.yaml:1]: don't think we want images from main

**Review by chrisahl — CHANGES_REQUESTED** [review]
Surprised you did updates to AWS!  I had talked to Zack about having some sort of tool to ease the manual work but we never got around to it

File: images/gcp/vm_images.yaml (hunk 1 above)
Comment (leena-jawale) [reply images/gcp/vm_images.yaml:1]: yes it is 2.4 image just have main name in there.. I'll rebuild

**Comment by leena-jawale** [comment]
> Surprised you did updates to AWS! I had talked to Zack about having some sort of tool to ease the manual work but we never got around to it

AWS changes came automatically. I think I'll close this PR and create another PR

**Pull request closed without merging**